packages/office-addin-manifest/src/manifestHandler/manifestHandler.ts

```diff
@@ -7,21 +7,20 @@ import * as fs from "fs";
 
 export abstract class ManifestHandler {
   /* eslint-disable no-unused-vars */
-  constructor() {
-    this.manifestPath = "";
+  constructor(manifestPath: string) {
+    this.manifestPath = manifestPath;
     this.fileData = "";
   }
 
-  public async readFromManifestFile(manifestPath: string): Promise<string> {
```

Comment:
Any particular reason for the name change?

Reply:
I don't think there is a strong reason to, I will just keep the name the same